server/src/app.js

```diff
@@ -25,7 +25,7 @@ express.response.error = function (error, statusCode) {
 app.use(require('morgan')('dev')) // logging
 app.use(require('helmet')({ contentSecurityPolicy: false }))
 app.use(require('compression')())
-app.use(formidable({ multiples: true }))
+app.use(formidable({ multiples: true, maxFileSize: 1024 ** 3 }))
```

Comment:
What does the ** 3 do?

Reply:
`**` is JS exponent operator, so 1024 * 1024 * 1024